process/project-lifecycle-documents/model_signing_sandbox_stage.md

```diff
@@ -0,0 +1,97 @@
+## Application for creating a new project at Sandbox stage
+
+### List of project maintainers
+
+The project has 4 maintainers from 3 different organizations:
+
+*  Laurent Simon, Google, @laurentsimon
+*  Daniel Major, NVidia,
+*  Eoin Wickens, HiddenLayer, @EWickens
+*  Mihai Maruseac, Google, @mihaimaruseac
+
+### Mission of the project
+
+The project must be aligned with the OpenSSF mission and either be a novel
+approach for existing areas, address an unfulfilled need, or be initial code
+needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF
+projects collaborate with the existing project rather than seek a new project.
+
+> Create a cryptographic signing specification for artificial intelligence and
+> machine learning models, addressing challenges such as very large models that
+> can be used separately, and the signing of multiple disparate file formats
+> held within a directory. This specification may have wider applicability to
+> signing directories of multiple arbitrary file formats. This specification may
+> later be proposed as a formal standard.
+
+#### Specific Goals Include:
+
+* Develop standards for efficient hashing of large models
+* Develop standards for efficient verification of models that contain multiple
+  formats in the same place
+* Develop standards for efficient verification of models at inference time.
+
+To achieve these goals, we work on developing
+https://github.com/sigstore/model-transparency library as an OSS standard for ML
+models signing and verification. The aim of this project is to guide the
+`model-transparency` development and help in standardizing hashing,
+verification, and deployment (e.g., model signature format).
+
+#### Non-Goals Include:
+
+* Developing a new model format
+* Handling security of ML outputs
```

Comment:
I'm assuming Assertions/verification of training data quality/bias/etc. or the training pipeline are outside of the scope? If there are other major non-goals, I suggest adding those, too.

Reply:
Great idea, thank you! Added this as a bullet and made it explicit that anything outside of model integrity is out of scope